legal-api/tests/unit/models/test_filing.py

```diff
@@ -25,20 +25,27 @@
 import pytest
 from flask import current_app
 from freezegun import freeze_time
+from legal_api.models import Business, Filing, User  # noqa: I001
+from legal_api.exceptions import BusinessException  # noqa: I001
 from registry_schemas.example_data import (
+    ALTERATION_FILING_TEMPLATE,
     ANNUAL_REPORT,
     CHANGE_OF_DIRECTORS,
     CORRECTION_AR,
+    COURT_ORDER,
     FILING_HEADER,
     SPECIAL_RESOLUTION,
 )
 from sqlalchemy_continuum import versioning_manager
 
-from legal_api.exceptions import BusinessException
```

Comment:
why were these moved?
the linter should complain about the move

Reply:
I don't know why but my Flake8 local is slightly different from the GH's action. I moved because Flake8 complianed. How can I make it use the same rules?

Reply:
are you on Windows+WSL or Mac/Linux? running ```make setup``` should get it all install correctly
if you are not, then make sure you do a ```pip install -e .``` in the services home directory, so it knows these modules are local and not external

Reply:
I'm using WSL 2. I used to run ```make setup``` but I also used to run ``` pip install .```. I see now that I cannot run pip after running make. I fixed the import. The strange is that the checks on github passed. Thanks!